lib/Doctrine/MongoDB/Event/AggregateEventArgs.php

```diff
@@ -63,4 +63,14 @@ public function getOptions()
     {
         return $this->options;
     }
+
+    public function setPipeline(array $pipeline)
```

Comment:
Even though the rest of the methods don't have docblocks, I think we can improve that one and add docblocks to the event args classes. Would you mind doing that?